server/sonar-web/src/main/js/apps/quality-gates/components/Details.js

```diff
@@ -80,9 +81,15 @@ export default class Details extends React.PureComponent {
     const { qualityGate, onSetAsDefault, onUnsetAsDefault } = this.props;
 
     if (qualityGate.isDefault) {
-      unsetQualityGateAsDefault(qualityGate.id).then(() => onUnsetAsDefault(qualityGate));
+      unsetQualityGateAsDefault(qualityGate.id).then(
+        () => onUnsetAsDefault(qualityGate),
+        throwGlobalError
```

Comment:
_This is just a notice for future._ We need to be consistent with this `throwGlobalError`: I add it in api, you add it in components. I'm fine with both solutions, but we need to have only one way.